tide_landing_page.install

```diff
@@ -776,3 +776,27 @@ function tide_landing_page_update_8018() {
     $field->save();
   }
 }
+
+/**
+ * Enable entity type/bundles for use with scheduled transitions.
+ */
+function tide_landing_page_update_8019() {
```

Comment:
@MdNadimHossain Were you going to add this in the install hook as well?

Reply:
@GROwen Does it needs to be in the install hook as well? Just trying to understand, if that the case, then isn't it the other update hooks should go into the install hook! isn't this something that goes when cleaning up all the hooks?